### PR TITLE
Add category

### PR DESCRIPTION
**Pull request by Matthew-Abebe**
Screenshot (774)
Screenshot (776)
Screenshot (775)

### CODE REVIEW

File: client/pages/purchase-form.jsx [hunk 1]
```diff
@@ -56,9 +71,10 @@ class PurchaseForm extends React.Component {
           <label>Enter Category</label>
           <select className="form-select" aria-label="Default select example" required value={categoryValue} onChange={this.handleCategoryInputChange}>
             <option value="" disabled hidden>Select an option</option>
-            <option value="Groceries">Groceries</option>
+            {/* <option value="Groceries">Groceries</option>
             <option value="Rent">Rent</option>
-            <option value="Car">Car</option>
+            <option value="Car">Car</option> */}
```
Comment (thebearingedge) [inline client/pages/purchase-form.jsx:76]: Please delete commented out code.

File: server/public/styles.css [hunk 2]
```diff
@@ -90,11 +94,27 @@ ul {
...
 
+.category-input-group {
+  position: absolute;
+  display: flex;
+  flex-flow: column wrap;
+  width: 50%;
+  margin: 0 auto;
+  left: 0;
+  right: 0;
+  top: 25%;
```
Comment (thebearingedge) [inline server/public/styles.css:105]: you shouldn't need absolute positioning on this.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: server/public/styles.css (hunk 2 above)
Comment (Matthew-Abebe) [reply server/public/styles.css:105]: Is it okay if I use fixed?

**Comment by Matthew-Abebe** [comment]
Screenshot (778)
Screenshot (779)

**Comment by Matthew-Abebe** [comment]
Screenshot (781)
Screenshot (782)

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**